instant-markdown-d

```diff
@@ -1,10 +1,10 @@
-#!/bin/sh
-':' //; exec "`command -v nodejs || command -v node`" "$0"
+#!/usr/bin/env node
```

Comment:
Are you positive that the original lines don't work? According to http://unix.stackexchange.com/users/46546/bruno-jouhier it should...